app/Http/Controllers/AuthorController.php

```diff
@@ -25,5 +25,26 @@ public function store(Request $request)
 
         return response()->json(['id' => $author->id]);
     }
+    public function destroy($id)
+    {
+
+        Author::destroy($id);
+        /*
+        !!No se encuentran los metodos!!
+        $author= Author::find($id);
+        $author->delete();
+        */
+
+
+        return "ok";
```

Comment:
Can we return something like https://github.com/avgvsto/moreno/blob/d1216256d86e1c999fdd320961e3c326a9407433/app/Http/Controllers/AuthorController.php#L37
?